pkgs/applications/misc/dunst/default.nix

```diff
@@ -15,11 +16,13 @@ stdenv.mkDerivation rec {
     sha256 = "102s0rkcdz22hnacsi3dhm7kj3lsw9gnikmh3a7wk862nkvvwjmk";
   };
 
+  patches = [ ./support-svg.patch ];
```

Comment:
Please use `fetchpatch` to download the patch from https://github.com/knopwob/dunst/commit/63b11141185d1d07a6d12212257a543e182d250a.patch instead of copying it into nixpkgs.